pkgs/servers/pulseaudio/default.nix

```
@@ -59,7 +59,7 @@ stdenv.mkDerivation rec {
     ./0007-Fix-link-args-on-darwin.patch
   ];
 
-  outputs = [ "out" "dev" ];
+  outputs = [ "out" "dev" "config" ];
```

```suggestion
  outputs = [ "out" "dev" "config-header" ];
```

I think this would be better

---

Done